### PR TITLE
Make history op screen much more minimal

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
image

image

### CODE REVIEW

File: apps/daimo-mobile/src/view/shared/time.ts [hunk 1]
```diff
@@ -13,5 +13,11 @@ export function timeAgo(sinceS: number, nowS: number) {
...
 export function timeString(s: number) {
   const date = new Date(s * 1000);
-  return date.toLocaleString();
+  return date.toLocaleString([], {
+    year: "numeric",
+    month: "numeric",
+    day: "numeric",
+    hour: "2-digit",
+    minute: "2-digit",
+  });
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/shared/time.ts:22]: drops seconds from display

File: apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx [hunk 2]
```diff
@@ -75,32 +64,46 @@ function LinkToExplorer({ txHash }: { txHash: string }) {
...
-  return (
-    <View style={styles.kvRow}>
-      <View style={styles.kvKey}>
-        <TextLight numberOfLines={1}>{k}</TextLight>
-      </View>
-      <View style={styles.kvVal}>
-        <TextBody numberOfLines={1}>
-          <TextBold>{v}</TextBold>
-        </TextBody>
+      <Spacer h={16} />
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx:91]: may be worth putting space around the title

File: apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx [hunk 3]
```diff
@@ -113,20 +116,12 @@ const styles = StyleSheet.create({
...
-  kvRow: {
-    flexDirection: "row",
-    alignItems: "center",
-    gap: 16,
-  },
-  kvKey: {
-    flex: 0,
-    width: 64,
+  h2Small: {
+    fontSize: 20,
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx:120]: regular h2 too big?

**Review by dcposch — APPROVED** [review]
left comments. either way, 🚢

File: apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx (hunk 2 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx:91]: doubled to 32

File: apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx (hunk 3 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx:120]: yeah and bold

**Pull request merged**